apps/server/src/modules/system/testing/system-entity.factory.ts

```diff
@@ -9,7 +11,7 @@ export const systemOauthConfigEntityFactory = BaseFactory.define<OauthConfigEnti
 	() => {
 		return {
 			clientId: '12345',
-			clientSecret: 'mocksecret',
+			clientSecret: AesEncryptionHelper.encrypt('mocksecret', Configuration.get('AES_KEY') as string),
```

Comment:
Note sure I understand the need for this change.

Small note: Configuration.get('AES_KEY') does not have a default value in default.schema.json
it has a value in test.json and development.json

Reply:
I guess it is wanted that there is no default for AES_KEY in default.schema.json. It is the same for other secrets, e.g. S3_KEY. Or do you see a problem with it.